bootstrap/bootstrap.ps1

```diff
@@ -90,7 +90,7 @@ if ($env:buildtools_source -ne $null)
 }
 $packagesPath = Join-Path $RepositoryRoot "packages"
 $dotNetExe = Join-Path $cliLocalPath "dotnet.exe"
-$restoreArgs = "restore $projectJson --packages $packagesPath --source $buildToolsSource --source $nugetOrgSource"
+$restoreArgs = "restore $projectJson --packages $packagesPath --source $buildToolsSource --source $nugetOrgSource $($env:init_tools_restore_args)"
```

Comment:
Do you mean you would change the name? (I prefer `BOOTSTRAP_RESTORE_ARGS` to match filename.)

The main point of this variable is https://github.com/dotnet/buildtools/blob/7f31a1d788492615f03ab5b9337ed1c72028edd0/src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh#L12-L13, so this could be changed to a positional arg in `init-tools.sh/cmd` (since it doesn't have "proper" arg handling) and an optional param in `bootstrap.sh/ps1` that passes it through.